### ISSUE TITLE
[BUG] Histogram Module is failing

### ISSUE THREAD

**Issue by ataata107** [publiclab/image-sequencer#1457]
### Please describe the problem (or idea)

![ezgif com-video-to-gif (5)](https://user-images.githubusercontent.com/32903329/72210771-9713c880-34e6-11ea-9975-949200145d4b.gif)

> What did you expect to see that you didn't?
Expected for the histogram to load

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by blurry-x-face** [comment]
Hi, This is working fine in my machine

![Screenshot from 2020-01-12 03-02-34](https://user-images.githubusercontent.com/42492389/72210899-11911800-34e8-11ea-8e55-32929ef5b03b.png)

**Comment by ataata107** [comment]
@blurry-x-face It is even not working on the beta site. Can you confirm that?

**Comment by ataata107** [comment]
@publiclab/is-reviewers

**Comment by blurry-x-face** [comment]
Just checked, you're right it is not working on beta site maybe something got updated on recent PRs, My version is maybe 2-3 commits behind.
Was changePixel method changed in any of the recent PRs?

**Comment by keshav234156** [comment]
@ataata107 @blurry-x-face yes there is a bug but fix doesn't seem to be correct. It's just a workaround solution @HarshKhandeparkar it's caused due to your PR . Can you please take a look at what went wrong and fix it.

**Comment by HarshKhandeparkar** [comment]
Taking a look rn.
